shell/probe/nocturnal.sh

```diff
@@ -1,30 +1,32 @@
 #!/bin/bash
 
-PRELUDE_API=${PRELUDE_API:="https://api.preludesecurity.com"}
+PRELUDE_SLEEP=${PRELUDE_SLEEP:=14440}
+PRELUDE_DIR=${PRELUDE_DIR:="./cache"}
+PRELUDE_CA=${PRELUDE_CA:="prelude-account-prod-us-west-1.s3.amazonaws.com"}
 
-sys=$(uname -s)-$(uname -m)
-dos=$(echo $sys | tr '[:upper:]' '[:lower:]')
+api="https://api.preludesecurity.com"
```

Comment:
pleeease keep the ability to override for staging

Reply:
downside is someone can use an env var to override where tests are coming from, so I'll likely opt not to keep that in.

Reply:
isn't that what the CA check is for?

Reply:
sort of... two different things.. but the CA check may override the concern as you're pointing out... I think you're spot on

Reply:
why'd this go back :(

Reply:
can't leave API and CA both mutable